scripts/strings.py

```diff
@@ -0,0 +1,32 @@
+#
+# strings.py - print the unique strings of runtime function arguments and return values.
+#
+# uftrace-option: --nest-libcall --auto-args
+#
+
+strset = set()
+
+def uftrace_entry(ctx):
+    global strset
+    if "args" in ctx:
+        args = ctx["args"]
+        for arg in args:
+            if isinstance(arg, str):
+                arg = arg.rstrip().lstrip()
```

Comment:
Simply `arg.strip()` won't work?

Reply:
Right. I thought that it also removes internal whitespaces. Will change it.